react/src/components/Chat/ChatCard.tsx

```diff
@@ -308,11 +341,15 @@ const ChatCard: React.FC<ChatCardProps> = ({
           // others
           fetchKey={fetchKey}
           closable={closable}
-          onClickCreate={onCreateNewChat}
+          clonable={clonable}
+          onClickCreate={() => {
+            onCreateNewChat?.(chat);
+          }}
           onClickClose={() => {
             onRequestClose?.(chat);
           }}
-          onClickDeleteChatHistory={() => {
+          onClickClearChatMessages={() => {
+            onClickClearChatMessages?.(chat);
             setMessages([]);
```

Comment:
We still need to call setMessage manually to clear chat message. useChat manage own message cache inside